src/main/antlr4/com/zendesk/maxwell/schema/ddl/mysql.g4

```diff
@@ -59,7 +71,8 @@ drop_column: DROP COLUMN? old_col_name;
 alter_rename_table: RENAME (TO | AS) table_name;
 
 convert_to_character_set: CONVERT TO charset_token charset_name collation?;
-default_character_set: DEFAULT? charset_token '=' charset_name ( COLLATE '=' (IDENT | STRING_LITERAL) )?;
+default_character_set: DEFAULT? charset_token '=' charset_name;
+default_collate: DEFAULT? COLLATE '=' (IDENT | STRING_LITERAL)?;
```

Comment:
The mysql page says COLLATE can be used only with charset change..

```
  | CONVERT TO CHARACTER SET charset_name [COLLATE collation_name]
  | [DEFAULT] CHARACTER SET [=] charset_name [COLLATE [=] collation_name]
```
Also = is optional.